layout-webapp/src/main/webapp/vue-app/common-layout-components/components/manage-permissions/ManagePermissionItem.vue

```diff
@@ -22,7 +22,7 @@
       size="19"
       class="fas fa-users me-3" />
     <span
-      class="d-flex text-truncate font-weight-bold text-start text-color body-2 mt-2">
+      class="d-flex text-truncate font-weight-bold text-start text-body mt-2">
```

Comment:
Same as https://github.com/Meeds-io/layout/pull/157/files/1207178a8889f8beb481a69a013b2027280937dd#r1668522608

```suggestion
      class="d-flex text-truncate font-weight-bold text-start mt-2">
```